report.php

```diff
@@ -171,6 +171,9 @@
     }
 
     if (!empty($formelements->datefrom)) {
+        if ($formelements->datefrom == $formelements->dateto) {
+            $formelements->dateto = $formelements->dateto + DAYSECS;
+        }
```

Comment:
should this not be `$formelements->dateto + DAYSECS - 1`, so that you end up with 23:59:59?

Reply:
I will change it.